packages/palette/src/elements/EntityHeader/EntityHeader.tsx

```diff
@@ -52,7 +52,13 @@ export const EntityHeader: SFC<EntityHeaderProps> = ({
       )}
 
       <Flex flexDirection="column" justifyContent="center" width="100%">
-        <Serif size="3" weight="semibold" color="black100">
+        <Serif
```

Comment:
Are these changes meant to be included?

(Also, they look like they belong in the `ios` version of this component, but I think that change was already made).

Reply:
oh whoops!

Reply:
Cleaned this up